src/css/theme/_search-box.scss

```diff
@@ -0,0 +1,30 @@
+.ais-search-box {
+  display: inline-block;
```

Comment:
How hard would it be to implement the search box like in ris? http://shipow.github.io/searchbox/